src/govuk/components/notification-banner/template.test.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const axe = require('../../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../../lib/jest-helpers')
+
+const examples = getExamples('notification-banner')
+
+describe('Notification-banner', () => {
+  describe('default example', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('notification-banner', examples.default)
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('aria-labelledby attribute matches the title id', () => {
```

Comment:
Being fussy, but it'd be good to fix this so it reads as a sentence.

```suggestion
    it('has an aria-labelledby attribute matching the title id', () => {
```